### DISCUSSION TITLE
Xc-token expiry duration [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2866] [Discord Threads]
> 🙋‍♂️ **windy**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@608112067014295572> 10 hours. You can configure it using environment variable `NC_JWT_EXPIRES_IN`    
####    
#### Details here (https://docs.nocodb.com/getting-started/installation#environment-variables)  
#  
> 🙋‍♂️ **windy**  
#### Can I set it not to expire ？  
#  
> 💁 **o1lab**  
#### xc-auth token had 10 hrs expiry  
#  
> 💁 **o1lab**  
#### xc-token doesn't have an expiry  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999223805685796874)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
